components/shared/setup-space.tsx

```diff
@@ -49,6 +49,8 @@ const SetupSpace = ({
         you to it. 👇
       </p>
       <form action={actionHref} onSubmit={() => handleSubmit} className="mb-8">
+        <input type="hidden" name="focusBgColor" value="#6673FF" />
+        <input type="hidden" name="backgroundColor" value="#3B2FC9" />
```

Comment:
It doesnt need to be dynamic, but since both funneled to the createSpace call, the thought was to be safe here and make it dynamic.

Reply:
Not following on how this works for the two views, is the value being set somewhere else? Seems like hardcoding the value would be wrong for one of the flows

Reply:
The part I'm not getting yet is how `setup-space` being used in two places has hardcoded (one) value. Should it not be dynamic so that the two views using it can dynamically set that value? Otherwise either onboarding and filefeed would have the same color scheme since it's hardcoded.

I think this is working correctly but I'm not understanding the code path yet